code/modules/mob/living/simple_animal/borer/borer_html.dm

```diff
@@ -0,0 +1,129 @@
+
+/mob/living/simple_animal/borer/proc/get_html_template(content)
+	var/html = {"<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\" \"http://www.w3.org/TR/html4/loose.dtd\">
+		<html>
```

Comment:
Hard to say that when we already have interfaces that aren't tgui, like the safe